src/call/github-events-to-s3.ts

```diff
@@ -19,6 +19,7 @@ export default async function githubEventsToS3(app: Probot, context: any, resour
   if (!(await validateResourceConfig(app, context, resource))) return;
 
   const repoName = context.payload.repository?.name;
+  const eventName = context.payload.action === undefined ? context.name : `${context.name}.${context.payload.action}`;
 
   const now = new Date();
   const [day, month, year] = [now.getDate(), now.getMonth() + 1, now.getFullYear()].map((num) => String(num).padStart(2, '0'));
```

Comment:
Seems like this is not properly tested?

Reply:
@bshien can you please share your research details on events like `gollum`?

Reply:
![Screenshot 2024-10-09 at 3 27 50 PM](https://github.com/user-attachments/assets/83c99b40-d54c-4353-8d9a-cf8b19b2beb4)
Some events like `gollum` don't have actions so it shows up like this

Reply:
Let me try to add tests for this

Reply:
I kept getting weird errors writing the tests because there wasn't this line `jest.clearAllMocks();` to clean up from previous tests. I've added it to the other test files, the tests still pass, let me know if this was the right thing to do.

Reply:
There is no

afterEach(() => {
  jest.clearAllMocks();
});

or 

afterEach(() => {
  jest.resetAllMocks();
});

?

Reply:
Added the afterEachs to the test files

Reply:
Thanks.